junseungbin/ProgrammersList/ProgrammersList.java

```diff
@@ -0,0 +1,81 @@
+package assignment;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+class ProgrammersList {
+    public static void main(String[] args) {
```

Comment:
아쉽게도 문제에서 지정한 조건과 조금 다르게 작성된 것 같아요 😢😢
프로그래머스에서는 아래와 같이 메서드의 형태가 고정되어 있답니다!

```suggestion
    public int[] solution(int n, int[] slicer, int[] num_list)
```